mitodl/micromasters
# implement past course run display

#### What are the relevant tickets?

this closes #3049 

#### What's this PR do?

this implements one part that was left off of the dashboard redesign PR to make it a little more manageable, the display of past course runs.

#### How should this be manually tested?

Basically, load up a bunch of different course runs and make sure the display of past runs makes sense!

You need to click on the `<Grades />` component to get the course run popup to show itself.
